google/cloud/forseti/scanner/scanners/data_models/cai_data_model.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Cloud Asset Inventory (CAI) Data Model."""
+
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.scanner.scanners.data_models import base_data_model
+from google.cloud.forseti.services.model.importer import importer
+
+LOGGER = logger.get_logger(__name__)
+
+CAI_RESOURCE_TYPE_MAPPING = {
```

Comment:
Is this repeated from above, from `config_validator_util/cv_data_converter.py`?  If so, can we consolidate or centralize it?

Reply:
Good catch- done!